http/bench/dune

```diff
@@ -2,10 +2,10 @@
  (name bench)
  (libraries http core core_bench))
 
-(rule
- (alias runtest)
- (package http)
- (enabled_if
-  (= %{ocaml-config:word_size} 64))
- (action
-  (run ./bench.exe -quota 5s)))
+; (rule
+;  (alias runtest)
+;  (package http)
+;  (enabled_if
+;   (= %{ocaml-config:word_size} 64))
+;  (action
+;   (run ./bench.exe time cycles)))
```

Comment:
I kept this commented to hint at how to run it

Reply:
I'll just attach it to a separate alias.

Reply:
Alternatively, you can also make the argument `-quota 1x`. This way it will not slow down the test suite.  We should also discard the stdout as it's garbage in this case.

Reply:
There is no point in compiling it and running it, I have added a `runtest` alias.

I am going to follow up with the optional c`ohttp-bench` package in the next few days and move everything there, including the latency benchmarks for `cohttp-{lwt,async}-server`. It is convenient to keep it at hand and will get rid of a lot of unnecessary test dependencies in the `http` package